templates/components/products/price-range.html

```diff
@@ -1,3 +1,35 @@
 {{#with price_range}}
-    {{min.formatted}} - {{max.formatted}}
-{{/with}}
+    {{#and price_range.min.with_tax price_range.max.with_tax}}
```

Comment:
since we're using a `{{#with price_range}}` above we can do `{{#and min.with_tax max.with_tax}}` instead and `{{#and}}` seems fine to me

Reply:
Thanks, will update